support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -94,48 +94,46 @@ export const inThreeTier = threeTierCheckoutEnabled(
 	commonState.amounts,
 );
 
+// Lazy load your components
+// const ThreeTierLanding = lazy(() => import('./twoStepPages/threeTierLanding'));
+const SupporterPlusCheckout = lazy(
```

Comment:
Presume these are the wait to load when I route to them

Reply:
Exactly!